server/services/SsotToRobotParsing/generateCodeForRpaTasks.js

```diff
@@ -44,11 +43,21 @@ const appendRpaInputParameter = (parameterObject) => {
     (a, b) => a.index - b.index
   );
   sortedInputs.forEach((parameter) => {
-    // regex will return -1 if no $$text$$ was found
-    newCodeLine +=
-      parameter.value.search(/\$\$(.*?)\$\$/) < 0
-        ? `${FOURSPACE}${parameter.value}`
-        : `${FOURSPACE}$\{${parameter.value.split('$$')[1]}}`;
+    if (parameter.requireUserInput && parameter.value === '') {
```

Comment:
Why not set newCodeLine = FOURSPACE at the beginning? Then you wouldn't need to set it every time.